src/edu/ucsb/nceas/metacat/util/AuthUtil.java

```diff
@@ -475,5 +494,33 @@ private static boolean onAccessList(Vector<String> accessList, String username,
 		}
 		return false;
 	}
+	
+	/**
+     * Convert a delimited string list to a Vector
+     * @param list  the list of delimited vectors
```

Comment:
"The string to be split into components."  Note that 'list' is not a great name for this parameter.  Maybe it would be better named "string" or "text"?